src/components/wallet/earn/Validate/RegisterNode.vue

```diff
@@ -158,17 +158,15 @@ export default class RegisterNode extends Vue {
             )
             console.log(result)
             this.$emit('registered')
-            await this.$store.dispatch('Notifications/add', {
+            this.helpers.dispatchNotification({
+                message: this.$t('notifications.register_node_success'),
                 type: 'success',
-                title: 'Node Registered',
-                message: 'Your node registered successfully.',
             })
         } catch (error) {
             console.error(error)
-            await this.$store.dispatch('Notifications/add', {
+            this.helpers.dispatchNotification({
+                message: this.$t('notifications.register_node_failed'),
                 type: 'error',
-                title: 'Registering Node Failed',
-                message: error,
```

Comment:
Not having a place for `error` is not a good thing. It loses some transparency about what went wrong, there can be multiple different reasons for an error message

Reply:
ill do a new refactor for the notifications to have an error message and a custom message its another task